third_party/move/move-compiler-v2/tests/uninit-use-checker/v1-commands/use_before_assign.exp

```diff
@@ -19,10 +19,10 @@ fun <SELF>_0::main() {
 
 Diagnostics:
 error: use of unassigned local `x`
-  ┌─ tests/uninit-use-checker/v1-commands/use_before_assign.move:4:9
+  ┌─ tests/uninit-use-checker/v1-commands/use_before_assign.move:4:13
   │
 4 │     let y = x;
-  │         ^
+  │             ^
```

Comment:
Bonus fix: looks like you can close https://github.com/aptos-labs/aptos-core/issues/11599 as well!

Reply:
Ah, great.

In some cases the new treatment is better, in others not. That is one of the better ones.